sycl/include/sycl/types.hpp

```diff
@@ -719,19 +641,21 @@ template <typename Type, int NumElements> class vec {
 #if defined(__SYCL_DEVICE_ONLY__)
     using OpenCLVecT = OpenCLT __attribute__((ext_vector_type(NumElements)));
     using OpenCLVecR = OpenCLR __attribute__((ext_vector_type(NumElements)));
+
+    auto extVecType = detail::BitCast<DataType, vector_t>(m_Data);
```

Comment:
`extVecType` is a bad name. I initially thought it's a type alias and not a variable.